### DISCUSSION TITLE
Table name containing character : (colon) causes Code completion not working

### DISCUSSION THREAD

**Discussion by olivier-75** [dbeaver/dbeaver#13986] [Q&A] [answered]
Dbeaver 5.2.2 with HANA.
Please include character :  (colon) in valid characters for table name. In our HANA project, lots of tables contains this character, and so code completion for columns is currently not working on those tables.

**Comment by eirikwang** [comment]
Same applies for spaces and $. 
Bothe code completion for columns and table names breaks.

**Comment by kai-morich** [comment, marked as answer]
: and other special characters are not part of standard identifiers, such identifiers have to be quoted.

Workaround: if you start the identifier with a quote (like `SELECT * FROM "T`) then autocompletion also support these special characters
